### DISCUSSION TITLE
[Question][Hw6]Blocks with No Transactions

### DISCUSSION THREAD

**Discussion by shyngys-aitkazinov** [prosyslab-classroom/cs348-information-security#636] [Q&A]
A have clarification question, in the problem, there some transactions, but Alice decides to mine empty block, right?

**Comment by minsukk-at-kaist-ac-kr** [comment]
Yes, Alice decides to mine a block with no transactions (i.e., empty block) even though she knows that there exist some transactions waiting to be included in a block.
